application/models.py

```diff
@@ -1,3 +1,115 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+# defines model for a single step (or box) from prescribing guide flowchart
+class Step(models.Model):
+    # Fields
+    name = models.CharField(help_text='Enter step title', max_length=25)
+    description = models.TextField(help_text='Enter step description')
+    
+    # Metadata
+    class Meta: 
+        ordering = ['name']
+
+    # Methods
+    def get_absolute_url(self):
+        """Returns the url to access a particular instance of MyModelName."""
+        return reverse('model-detail-view', args=[str(self.id)])
+    
+    def __str__(self):
+        """String for representing the MyModelName object (in Admin site etc.)."""
+        return self.name
+
+# defines model for disorder treatment type and corresponding steps
+class Treatment(models.Model):
+    # Fields
+    name = models.CharField(help_text='Enter disorder treatment type title', max_length=25)
+    steps = models.ManyToManyField(Step)
+    
+    # Metadata
+    class Meta: 
+        ordering = ['name']
+
+    # Methods
+    def get_absolute_url(self):
+        """Returns the url to access a particular instance of MyModelName."""
+        return reverse('model-detail-view', args=[str(self.id)])
+    
+    def __str__(self):
+        """String for representing the MyModelName object (in Admin site etc.)."""
+        return self.name
+
+# defines model for a medication used by prescribers
+class Medication(models.Model):
+    # Fields
+    name = models.CharField(help_text='Enter medication name', max_length=25)
+    category = models.CharField(help_text='Enter medication type', max_length=50)
+    initial_Dose = models.FloatField(help_text='Enter medication initial dose in mg')
```

Comment:
python variable names should be all lowercase -- initial_dose, maximum_dose etc.

Reply:
I just did that for display purposes, but I spose since that's on the admin page it doesn't actually matter; I'll change those back to convention